build.sbt

```diff
@@ -160,6 +161,14 @@ lazy val macrosAkka = projectMatrix
   .jvmPlatform(scalaVersions = scala2)
   .jsPlatform(scalaVersions = scala2)
 
+lazy val macrosPekko = projectMatrix
```

Comment:
this should be included in the root aggregate project list

Reply:
🤦‍♂️ I will add it

Reply:
Added.